github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1134,7 +1134,7 @@ periodics:
   # note: when the provider is updated, the `robots/cmd/perf-report-creator/graph.sh` and
   #       `robots/cmd/perf-report-creator/scrape.sh` needs to be updated as well, refer to this
   #       https://github.com/kubevirt/project-infra/pull/3032 for the needed changes
-  name: periodic-kubevirt-e2e-k8s-1.29-sig-performance
+  name: periodic-kubevirt-e2e-k8s-1.31-sig-performance
```

Comment:
@brianmcarey can you please also update the `robots/cmd/perf-report-creator/graph.sh` , ref: https://github.com/kubevirt/project-infra/pull/3032/

Reply:
Yes sure - sorry glanced over that in the comment.